iot/api-client/end_to_end_example/requirements.txt

```diff
@@ -0,0 +1,7 @@
+cryptography==2.1.3
+google-api-python-client==1.6.4
+google-auth-httplib2==0.0.3
+google-auth==1.2.1
+google-cloud==0.30.0
```

Comment:
Please don't pull in the broad google-cloud package, please include individual google-cloud-* packages that you actually need here.

Reply:
Done.